components/datatypes/financial-account.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/financial-account",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Financial Account",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Financial account details, such as owners, balances, and types.",
+  "definitions": {
+    "financialAccount": {
+      "properties": {
+        "xdm:financialAccountID": {
+          "title": "Financial Account ID",
+          "type": "string",
+          "description": "Current account balance for a single user account."
+        },
+        "xdm:financialAccountName": {
+          "title": "Financial Account Name",
+          "type": "string",
+          "description": "Name assigned to the financial account."
+        },
+        "xdm:financialAccountType": {
+          "title": "Financial Account Type",
+          "enum":["checking", "savings", "retirement", "investment"],
+          "description": "Describes the type of finanical account, such as checking, savings, retirement, or investment."
+        },
+        "xdm:currentAccountBalance": {
+          "title": "Current Financial Account Balance",
+          "type": "number",

```

Comment:
Can we use currency datatype here ? 
"https://ns.adobe.com/xdm/datatypes/currency" . The same comment could apply to all the amount fields which could have a different currency.